source/features/extend-status-labels.tsx

```diff
@@ -8,24 +8,33 @@ function init(): false | void {
 		return false;
 	}
 
+	// Get the latest status update
 	const lastActionRef = select.all(`
```

Comment:
This no longer a link to the `ref`, but just the last action. Perhaps `lastStatusChange` and you can drop the comment.

```suggestion
	const lastStatusChange = select.all(`
```

Reply:
Oh okay, thought the "ref" was some React-related convention thingy. 👍

Reply:
I think I had used `ref` in the git sense, like tags and commits are "refs"